src/plugins/data/public/query/query_string/dataset_service/dataset_service.ts

```diff
@@ -64,14 +63,11 @@ export class DatasetService {
 
   public async cacheDataset(dataset: Dataset): Promise<void> {
     const type = this.getType(dataset.type);
-    if (dataset) {
+    if (dataset && dataset.type !== DEFAULT_DATA.SET_TYPES.INDEX_PATTERN) {
       const spec = {
         id: dataset.id,
         title: dataset.title,
-        timeFieldName: {
-          name: dataset.timeFieldName,
-          type: 'date',
```

Comment:
i think i added type here for querying purposes.

Reply:
Is there any functionality I can test to see if changing this breaks something? The `timeFieldName` not being set properly is why time fields never worked when going through the configurator.